### PR TITLE
1. process_sales pipeline

### PR DESCRIPTION
**Pull request by lov3catch**
**Airflow DAGs:**

### bronze:
Do: prepare folder to store bronze data
Start_at: 2022-09-01, daily
Params: 
1. src - folder with customers raw data
2. dst - folder for bronze customers data
[DAG](https://github.com/lov3catch/data_engineering/pull/10/files#diff-73d5c51764c372791631816df1ba4622fbc6e67510242cd4ed553aa8d770c4cb)

### silver:
Do: call [spark job](https://github.com/lov3catch/data_engineering/pull/10/files#diff-ce12889f84336e6682123ddd3a90fe125bfedc0172665d8ebf2dfa34e61b714b)
Start_at: 2022-08-01, daily
Params: 
1. application - spark script location
[DAG](https://github.com/lov3catch/data_engineering/pull/10/files#diff-3fe82d6eafe0b873220131e2c408e00059347dae7d4ca205539980985bd46918)

**Spark**
Do: normalise purchase_date, price and product_name to lower case, normalise column names
Triggered by: [DAG](https://github.com/lov3catch/data_engineering/pull/10/files#diff-3fe82d6eafe0b873220131e2c408e00059347dae7d4ca205539980985bd46918)
Result: /app/file_storage/processed/silver/sales/sales.csv (partition)

WARNING: python & spark versions - must be equals.

### CODE REVIEW

File: lesson_17/dags/sales/bronze.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "src": Param(
+                "/app/file_storage/raw/sales/",
+                type="string",
+                title="Raw data is here",
+            ),
+            "dst": Param(
+                "/app/file_storage/processed/bronze/sales/",
+                type="string",
+                title="Processed data is here",
+            ),
```
Comment (DenysKulemza) [inline lesson_17/dags/sales/bronze.py]: краще винести в окрему змінну. Це коментар відноситься і для інших PR'ів

File: lesson_17/dags/sales/silver.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        schedule='@daily',
+        max_active_runs=1,
+        default_args=args,
+        params={
+            "application": Param(
+                '/app/tasks/sales_bronze_to_silver.py',
+                type="string",
+                title="Spark application",
+            ),
+        },
```
Comment (DenysKulemza) [inline lesson_17/dags/sales/silver.py]: теж саме

File: lesson_17/dags/sales/bronze.py (hunk 1 above)
Comment (lov3catch) [reply lesson_17/dags/sales/bronze.py]: Done

File: lesson_17/dags/sales/silver.py (hunk 2 above)
Comment (lov3catch) [reply lesson_17/dags/sales/silver.py]: Done

**Review by DenysKulemza — APPROVED** [review]

**Pull request merged**